plugins/modules/rulebook_activation.py

```diff
@@ -143,14 +154,18 @@
   state:
     description:
       - Desired state of the resource.
+      - The state of an activation itself is controlled by this parameter, whether enabled or
+        disabled.
+      - When an activation is created, it is enabled by default. Thus, present is equivalent to
```

Comment:
Since param `enabled` is deprecated, how can a user create an activation in disabled mode?
Your code does not force activation_param["is_enabled"] = True when user does not provide enable param but set state to "present".

Reply:
Yes, it goes with the backend defaults. But it's not a bad idea to enforce it to true it when state is present. I will document both scenarios.

Reply:
@kaiokmo I think for now it must be a two-operations thing because we internally discussed to postpone the wait for logic for later iterations. And this behavior would require the wait for logic since we need to disable the activation and wait to be disabled before editing it.